### ISSUE TITLE
Unable to load data: connection error: desc = "transport: authentication handshake failed: credentials: cannot check peer: missing selected ALPN property"

### ISSUE THREAD

**Issue by kzeroual** [argoproj/argo-cd#20121]
I can't understand why this error occurred.

![argocd issue](https://github.com/user-attachments/assets/ede7d6e2-c5d7-48c2-889f-3bd67e065f51)

**Comment by gerbenluimes** [comment]
I am having the same issue which seems to appear all of a sudden.

**Comment by serafdev** [comment]
Getting this too, I tried all browsers, upgraded argocd etc

**Comment by ahyar71** [comment]
same issue here.. please help

**Comment by gerbenluimes** [comment]
Also this thread on Slack https://cloud-native.slack.com/archives/C01TSERG0KZ/p1727190915361549

**Comment by ahyar71** [comment]
> same issue here.. please help

I just found out that argocd was auto-upgraded to version 2.14 on 24-09-2024. Anyone know what trigger this? since I don't have any action in my cluster when the event happens.

![image](https://github.com/user-attachments/assets/61a8009e-01bd-447a-8c5e-d9af7265a280)

**Comment by kzeroual** [comment]
I just solved this issue by changing the image tag of argocd-server from the latest version to v2.13.0-rc1

**Comment by ahyar71** [comment]
Yes, its solved by updating the argocd-server image. Thanks for the discussion

> I just solved this issue by changing the image tag of argocd-server from the latest version to v2.13.0-rc1

**Comment by A1EF** [comment]
I can't understand why issue has been closed. Still actually. I faced with same problem. Just replacing `latest` by `v2.12.6` and all works fine. What reason for use `quay.io/argoproj/argocd:latest`? Using `latest is always bad idea.

**Comment by leoluz** [comment]
Reopening as root cause isn't fixed yet in master

**Comment by A1EF** [comment]
Sorry for my fault: I downloaded file from master instead tagged branch.

**Comment by andrii-korotkov-verkada** [comment]
@crenshaw-dev, is this something ArgoCD should workaround, or should users update their setup?
